namd/src/GlobalMasterColvars.h

```diff
@@ -0,0 +1,26 @@
+#ifndef GLOBALMASTERCOLVARS_H
+#define GLOBALMASTERCOLVARS_H
+
+#include <memory>
+
+#include "colvarmodule.h"
+#include "colvarproxy.h"
+#include "colvarproxy_namd.h"
```

Comment:
In this file, only `GlobalMaster.h` needs to be included. You can forward declare `colvarproxy_namd`, and move the constructor to somewhere else, like a new `GlobalMasterColvars.C` or `colvarproxy_namd.C`.

Reply:
Indeed. I like the latter option better, since it's one fewer new file for a package that doesn't use CMake. There is a very similar init function in `colvarproxy_vmd.C`